### PR TITLE
Feature/create confirm workplace details base class

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- create confirm workplace details base class
- extend and reuse on registration and add-workplace flows

### CODE REVIEW

**Comment by omarCreativeDev** [comment]
[(5) Parent and subsidiaries - Add a workplace - Non - CQC Regulated](https://trello.com/c/bcbbFepb/200-5-parent-and-subsidiaries-add-a-workplace-non-cqc-regulated)

File: src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          {{ locationAddress.locationName }}
+        </dd>
+        <dd class="govuk-summary-list__actions">&nbsp;</dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Address
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <div *ngIf="locationAddress.addressLine1">{{ locationAddress.addressLine1 }}</div>
```
Comment (ChazUK) [inline src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html]: I feel like this should definitely be in the `address` element tag, dunno how that didn't get picked up by accessibility, but we can change it later.

File: src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+          <div *ngIf="locationAddress.addressLine2">{{ locationAddress.addressLine2 }}</div>
+          <div *ngIf="locationAddress.townCity">{{ locationAddress.townCity }}</div>
+          <div *ngIf="locationAddress.county">{{ locationAddress.county }}</div>
+          <div *ngIf="locationAddress.postalCode">{{ locationAddress.postalCode }}</div>
+        </dd>
+        <dd class="govuk-summary-list__actions">&nbsp;</dd>
```
Comment (ChazUK) [inline src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html]: Pretty sure you can omit the actions column if it's not used.

Comment (omarCreativeDev) [reply src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html]: tried to omit just now but the middle cell then doesn't span all the way, have left it in for now if thats ok

File: src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html (hunk 1 above)
Comment (omarCreativeDev) [reply src/app/features/add-workplace/confirm-workplace-details/confirm-workplace-details.component.html]: done

**Pull request merged**